examples/http_curl_command.rs

```diff
@@ -0,0 +1,22 @@
+//! This example demonstrates how to use the http curl command
```

Comment:
this file is to be deleted as discussed in the parent isuse

Reply:
Yup, using this as a temporary driver. Will delete before finishing up.